### PR TITLE
docs: Mention file extension for issue_template directory

### PR DESCRIPTION
**Pull request by s-hamann**
I noticed that [the docs on the issue templates directory](https://docs.gitea.io/en-us/issue-pull-request-templates/#issue-template-directory) do not mention the file extension requirement. This confused me and I had to dig in the code to find out, why my templates were not working.

This trivial PR addresses this.

### CODE REVIEW

File: docs/content/doc/usage/issue-pull-request-templates.en-us.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ Possible directory names for issue templates:
 - `.gitlab/ISSUE_TEMPLATE`
 - `.gitlab/issue_template`
 
-Inside the directory can be multiple issue templates with the form
+Inside the directory can be multiple issue templates with file extension `.md` and content of the form
```
Comment (delvh) [inline docs/content/doc/usage/issue-pull-request-templates.en-us.md]:
```suggestion
Inside the directory can be multiple markdown issue templates of the form
```

**Review by techknowlogick — APPROVED** [review]

**Review by delvh — APPROVED** [review]

File: docs/content/doc/usage/issue-pull-request-templates.en-us.md (hunk 1 above)
Comment (s-hamann) [reply docs/content/doc/usage/issue-pull-request-templates.en-us.md]: I think it's important to mention the allowed file extension explicitly.
Stating that the content is markdown gives a hint, but apparently there are other, less commonly used file extensions, such as `.mkd` or `.markdown`. Furthermore, one might still not make a connection between the content and the need for a specific file extension and try to use template files without any extension. I would probably have done that.
That being said, I'm open to suggestions on how to include that information in a more elegant way.

Comment (delvh) [reply docs/content/doc/usage/issue-pull-request-templates.en-us.md]:
```suggestion
Inside the directory can be multiple markdown (`.md`) issue templates of the form
```
?

Comment (s-hamann) [reply docs/content/doc/usage/issue-pull-request-templates.en-us.md]: Simple and effective. Thank you for the suggestion.

**Review by delvh — APPROVED** [review]

**Comment by 6543** [comment]
:rocket:

**Pull request merged**